coda.io.website.json

```diff
@@ -0,0 +1,31 @@
+{  
+    "providerId":"coda.io",
+    "providerName":"Coda",
+    "serviceId":"website",
+    "serviceName":"Coda Site",
+    "version": 1,
+    "syncPubKeyDomain" : "coda.io",
+    "logoUrl":"https://domainconnect.org/wp-content/uploads/2016/06/wix-300x103.png",
```

Comment:
This is incorrect, but wondering what the process is to get the proper logo uploaded to your CDN?
Will update this PR whenever that is done.

Reply:
Looks like these logos can come from any domain. The ones pointed at domainconnect are probably either using a default logo or are some kind of example domain

Reply:
Yeah. I'm in contact with the DomainConnect folks in Slack. Looks like there is a process to add the logos, just want to know what that entails.

<img width="803" alt="Screen Shot 2022-12-30 at 5 06 30 PM" src="https://user-images.githubusercontent.com/8244301/210114667-b03e3163-5293-4376-8278-80d6f58f0c5b.png">

Reply:
Added a logo from Coda's CDN